packages/protocol/migrationsConfig.js

```diff
@@ -118,18 +122,19 @@ const DefaultConfig = {
   reserve: {
     tobinTaxStalenessThreshold: HOUR, // 1 hour
     dailySpendingRatio: toFixed(0.05).toFixed(), // 5%
-    // TODO(Roman): Set these appropriately.
-    frozenGold: 0,
-    frozenDays: 0,
     spenders: [],
-    otherAddresses: [],
-    assetAllocationSymbols: ['cGLD', 'BTC', 'ETH'], // TODO(roman)
-    assetAllocationWeights: [0.5, 0.25, 0.25], // TODO(roman)
+    // To be populated once addresses are generated.
+    otherAddresses: ['0xd0a57D8acFe9979d33933d8A52971E6DC9E2DbF0'],
+    assetAllocationSymbols: ['cGLD', 'BTC', 'ETH', 'DAI'],
+    assetAllocationWeights: [0.5, 0.2, 0.1, 0.2],
   },
   reserveSpenderMultiSig: {
-    //Placeholder until addresses for 2/2 multsig are generated.
-    signatories: [`0x5409ed021d9299bf6814279a6a1411a7e866a631`],
-    numRequiredConfirmations: 1,
+    // 2/2 multsig
+    signatories: [
+      '0x49eFFA2ceF5FccA5540f421d6b28e76184cc0fDF',
+      '0x4550F1576fAC966Ac8b9F42e1D5D66D3A14dD8D3',
+    ],
+    numRequiredConfirmations: 2,
```

Comment:
Can we make both required confirmations be 2 for this?

Reply:
sure